src/coreclr/jit/emitxarch.cpp

```diff
@@ -14441,12 +14447,12 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
         case INS_shl_1:
         case INS_shr_1:
         case INS_sar_1:
-            result.insLatency = PERFSCORE_LATENCY_1C;
```

Comment:
With your change no latency is assign for IF_MRW, IF_SRW and IF_ARW

Reply:
Yes, I was following the example of other similar instructions, but then later saw that it's actually pretty inconsistent: #49647